src/FixedDataTable.js

```diff
@@ -957,12 +957,44 @@ var FixedDataTable = createReactClass({
       children.push(child);
     });
 
-    // Allow room for the scrollbar, less 1px for the last column's border
-    var adjustedWidth = props.width - Scrollbar.SIZE - Scrollbar.OFFSET;
+    var columns;
+    var columnGroups;
 
-    var useGroupHeader = false;
-    if (children.length && children[0].type.__TableColumnGroup__) {
-      useGroupHeader = true;
+    var tableWidth = props.width;
+    var useGroupHeader = children.length > 0 && children[0].type.__TableColumnGroup__ === true;
+
+    if (useGroupHeader) {
+      var columnGroupSettings =
+        FixedDataTableWidthHelper.adjustColumnGroupWidths(
+          children,
+          tableWidth
```

Comment:
won't using this here and below allocate too much flex if the vertical scrollbar is visible?